cliquet/resource/viewset.py

```diff
@@ -28,12 +28,27 @@ class ViewSet(object):
 
     readonly_methods = ('GET', 'OPTIONS', 'HEAD')
 
+    content_types = ["application/json"]
```

Comment:
One could want to accept xml for his service, that's the reason why I added this configuration here.